packages/discord-utilities/src/lib/limits.ts

```diff
@@ -137,7 +140,40 @@ export const MessageLimits = {
 	/**
 	 * Maximum characters allowed in a single message for a nitro user.
 	 */
-	MaximumNitroLength: 4000
+	MaximumNitroLength: 4000,
+
+	/**
+	 * Maximum numbers of reactions allowed for a message.
+	 */
+	MaximumReactions: 20,
```

Comment:
Out of all numbers, you chose 20?

Reply:
These are discord API limits. Not our own limits.

Reply:
You cannot add more than 20 different reactions to a message

Reply:
I was thinking something else. Ignore this :)